tests/test_main.py

```diff
@@ -659,6 +659,69 @@ def get_transform(augment):
     image, target, path = next(iter(train_ds))
     assert m.transforms.__doc__ == "This is the new transform"
 
+def test_config_augmentations():
+    """Test that augmentations can be configured via config."""
+    # Test with config args containing augmentations
+    config_args = {
+        "train": {
+            "augmentations": ["HorizontalFlip", "Downscale"]
+        }
+    }
+
+    m = main.deepforest(config_args=config_args)
+    csv_file = get_data("example.csv")
+    root_dir = os.path.dirname(csv_file)
+
+    # Load dataset with config-based augmentations
+    train_ds = m.load_dataset(csv_file, root_dir=root_dir, augment=True)
+
+    # Check that we can iterate over the dataset
+    image, target, path = next(iter(train_ds))
+    assert image is not None
+
+
+def test_config_augmentations_with_params():
+    """Test that augmentations with parameters can be configured via config."""
+    # Test with config args containing augmentations with parameters
+    config_args = {
+        "train": {
+            "augmentations": [
+                {"HorizontalFlip": {"p": 0.8}},
+                {"Downscale": {"scale_range": (0.5, 0.9), "p": 0.3}}
+            ]
+        }
+    }
+
+    m = main.deepforest(config_args=config_args)
+    csv_file = get_data("example.csv")
+    root_dir = os.path.dirname(csv_file)
+
+    # Load dataset with config-based augmentations
+    train_ds = m.load_dataset(csv_file, root_dir=root_dir, augment=True)
+
+    # Check that we can iterate over the dataset
+    image, target, path = next(iter(train_ds))
+    assert image is not None
+
+
+def test_config_no_augmentations():
+    """Test that default behavior works when no augmentations are specified in config."""
+    # Test with no augmentations in config (should use defaults)
+    m = main.deepforest()
+    csv_file = get_data("example.csv")
+    root_dir = os.path.dirname(csv_file)
+
+    # Load dataset - should use default augmentations
+    train_ds = m.load_dataset(csv_file, root_dir=root_dir, augment=True)
+
+    # Check that we can iterate over the dataset
+    image, target, path = next(iter(train_ds))
+    assert image is not None
+
+#TODO: Fix this test to check that predictions change as checking
+# if the threshold is changed in the config is probably not what
+# we actually want to test.
+@pytest.mark.xfail
```

Comment:
Still not clear what is happening with this change and how it relates to everything else.

Reply:
That's my guess. Looks like it's in the first commit so probably just didn't get cleaned out during @bw4sz's reset and recommit.